amf-webapi/jvm/src/main/scala/amf/plugins/document/webapi/validation/remote/JvmPayloadValidator.scala

```diff
@@ -175,6 +164,9 @@ case class JvmReportValidationProcessor(override val profileName: ProfileName,
       case e: InvalidJsonValue =>
         Seq(invalidJsonValidation("Invalid json value was provided", element, e))
 
+      case e: ArithmeticException if e.getMessage == "Division undefined" =>
```

Comment:
What other types of messages are there?

Reply:
there are more cases that throw ArithmeticExceptions that don't relate to dividing by zero such as:
- Overflowing of a data type when exact result is requested e.g. `Math.addExact(int x, int y)`
- Rounding on an operation that yields an inexact result e.g. `BigDecimal.ROUND_UNNECESSARY`

I've avoided catching these cases, as I want only the division by 0 errors to report an exact violation message. These other cases should occur, will fall onto the general catch I've added.

Reply:
excelent